svg-battery-indicator.el

```diff
@@ -38,19 +38,25 @@
   :group 'svg-battery-indicator
   :type 'integer)
 
+(defcustom svg-battery-indicator-stroke-face 'mode-line
+  "Face used for battery image borders."
+  :group 'svg-battery-indicator
+  :type 'face)
```

Comment:
If we do use a face here, I as a user would expect anything called \*-face which controls colouring to be a *face*, not variable pointing at a symbol for a face. So if we do use a face (which again, I'm not sure we should), it should be something more like:

``` emacs-lisp
(defface svg-battery-indicator-stroke '((t :inherit mode-line))
  "Face used for battery image borders."
  :group 'svg-battery-indicator)
```